.github/workflows/go.yml

```diff
@@ -37,3 +38,9 @@ jobs:
         name: binaries
         path: |
           bin/
+          
+    - name: Release
+      uses: softprops/action-gh-release@v1
```

Comment:
Recommend switching to `ncipollo/release-action`

Reply:
It would definitely be nice to avoid the deprecation warnings. There's a [PR to fix softprops/action-gh-release](https://github.com/softprops/action-gh-release/pull/266) although there hasn't been much activity on that repository recently so it will be interesting whether it gets merged or not.

How did you choose `ncipollo/release-action`?

Since this is a straightforward Go repository, [goreleaser-action](https://github.com/marketplace/actions/goreleaser-action) looks like a really interesting alternative, although I've not tried it.

Reply:
`ncipollo/release-action` was one of the top search results on GitHub Marketplace when I was looking for an action to handle releases and it had the most options / seemed the most flexible. I use it in the [Fabric CA release workflow](https://github.com/hyperledger/fabric-ca/pull/319/files#:~:text=%2D%20name%3A%20Release,%3A%20%24%7B%7B%20secrets.GITHUB_TOKEN%20%7D%7D).

Reply:
I saw this referenced in the PR to fix the other deprecation warning, so just wanted to plug you can use the vanilla `gh` CLI client too which I suspect will be very stable. Here's an example:
https://github.com/rubyatscale/use_packwerk/blob/main/.github/workflows/cd.yml

Reply:
Neat, I like that, thanks @alexevanczuk